lib/submit-analytics.js

```diff
@@ -23,6 +23,76 @@
 
 'use strict';
 
+const { resolve } = require('path');
+let utils = require('./utils');
+
+const nodeRequire = require;
+
+class WebhookProdDetails {
+    /**
+     * Path and credentials required to submit analytics through the webhook (production mode).
+     */
+    constructor() {
+        this.urlPrefix =
+                "https://webhooks.mongodb-realm.com/api/client/v2.0/app/realmsdkmetrics-zmhtm/service/metric_webhook/incoming_webhook/metric?ip=1&data=";
+        this.urlSuffix = "&ip=1";
```

Comment:
This seems redundant now that `?ip=1` is already in the `urlPrefix`?

```suggestion
```

Reply:
Fixed in #3524 .